go-1.19.yaml

```diff
@@ -0,0 +1,91 @@
+package:
+  name: go-1.19
+  version: 1.19.4
+  epoch: 0
+  description: "the Go programming language"
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: BSD-3-Clause
+  dependencies:
```

Comment:
Question: instead of having a `go` meta-package, should this `provides: go=1.19.4`?